cmake/UserOverride.cmake

```diff
@@ -17,21 +17,21 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
             set(common "${common} -DMS_WIN64")
         endif()
     endif()
-    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -ffast-math -funroll-loops")
-    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g")
+    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -march=native -ffast-math -funroll-loops")
+    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g -ggdb")
 elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Intel")
     # icpc
-    set(common "-std=c++11 -Wall -fno-common")
+    set(common "-Wall -fno-common")
     set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -xHOST -O3 -no-prec-div")
     set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g -O0")
 elseif (CMAKE_CXX_COMPILER_ID MATCHES Clang)
     # clang
-    set(common "-std=c++11 -Wall -Wextra -fPIC")
-    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -ffast-math -funroll-loops")
-    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g")
+    set(common "-Wall -Wextra -Wno-unused-parameter -fPIC")
+    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -march=native -ffast-math -funroll-loops")
+    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g -ggdb")
 elseif (CMAKE_CXX_COMPILER_ID STREQUAL "PGI")
     # pgcpp
-    set(common "--gnu --c++11 -D__GXX_EXPERIMENTAL_CXX0X__")
+    set(common "-D__GXX_EXPERIMENTAL_CXX0X__")
```

Comment:
This should be moved to CMakeLists.txt

Reply:
Fixed in 55c914574fa28074d73c1cc2547dcd94eeab84c0.